lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
Just concerned with people confusing that as a requirement, as in Cable requires the consumer be set in order to function. It also gives more of the impression that the recommended way to access is via a global property on Cable rather than by using something appropriate to your own JS framework.

That this is only generated when actioncable is installed, I don't think there is much risk of squashing existing JS, so its main purpose should be in leading newer developers in the right direction.